src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    public function relationsTabAction(ContentView $view)
+    {
+        $relationService = $this->container->get('ezsystems.platform_ui.hybrid.services.relation_service');
+        $contentTypeService = $this->getRepository()->getContentTypeService();
+        $versionInfo = $view->getContent()->getVersionInfo();
+
+        $relations = $relationService->loadAllRelations($versionInfo);
+
+        $contentType = $contentTypeService->loadContentType($versionInfo->getContentInfo()->contentTypeId);
+
+        $fieldNames = [];
+        foreach ($relations as $relation) {
+            $fieldDefinition = $contentType->getFieldDefinition($relation->sourceFieldDefinitionIdentifier);
+            $fieldNames[$relation->id] =
+                $fieldDefinition ? $fieldDefinition->getName($relation->getSourceContentInfo()->mainLanguageCode) : '';
+        }
+
+        $offset = null;
```

Comment:
I would expect pagination to be applied before loading, maybe that's not the EZ standard though?